nilearn/plotting/displays.py

```diff
@@ -889,6 +922,11 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
                 display_ax.draw_position(size=size, bg_color=bg_color,
                                          **kwargs)
 
+        if scalebar:
+            axes = self.axes.values()
+            for display_ax in axes:
```

Comment:
display_ax -> display_axis
Saving 2 letters, not worth the obfuscation.

Reply:
`display_ax` is being used many times in the previous code. I have no issue updating this instance, but that will make it inconsistent with the rest of the file.

Reply:
Yes, I don't mind it.